.gitignore

```diff
@@ -4,5 +4,4 @@ pkg/*
 .DS_Store
 Gemfile.lock
 coverage
-.history/
 .vscode/
```

Comment:
You've missed `.vscode` (you could amend the commit if you want to)

Reply:
sorry about that. Fixed and squashed commits